src/app/container/tool-file-editor/tool-file-editor.component.ts

```diff
@@ -113,7 +113,8 @@ export class ToolFileEditorComponent extends FileEditing {
           // Only stop editing when version change was successful (not 204)
           this.toggleEdit();
           this.containerService.setTool(editedDockstoreTool);
-          this.alertService.detailedSuccess();
+          const updatedVersion = this.getNewestVersion(editedDockstoreTool.workflowVersions);
+          this.alertService.detailedSuccess('Saved version ' + updatedVersion.name + ' of hosted entry ' + editedDockstoreTool.namespace);
```

Comment:
I would have said `tool`, even though Gary suggests `entry`. Why not be specific? You use `workflow` in the corresponding workflow component. But since he and I disagree, it's fine to leave it.

Reply:
It's because it adds no value to the user to be specific like that. However, there's a chance for developers to make mistakes like this (saying it's a workflow when it's not) which would confuse the user more. The same mistake can be made for services or any other entry type we may add in the future.